x-pack/solutions/security/plugins/security_solution/public/configurations/tabs/integrations.tsx

```diff
@@ -4,8 +4,81 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import * as React from 'react';
 
-export const Integrations = () => {
-  return <h1>{'Integrations'}</h1>;
-};
+import React from 'react';
+import { EuiSkeletonLoading } from '@elastic/eui';
+import type { AvailablePackagesHookType } from '@kbn/fleet-plugin/public';
+import { Routes, Route } from '@kbn/shared-ux-router';
+import { Redirect } from 'react-router-dom';
+import { AI4DSOC_ALLOWED_INTEGRATIONS, CONFIGURATIONS_PATH } from '../../../common/constants';
+import { useEnhancedIntegrationCards } from './hooks/use_enhanced_integration_cards';
+import { ConfigurationTabs, IntegrationsFacets } from '../constants';
+import { IntegrationsPage, IntegrationsSkeleton } from './components';
+import { withLazyHook } from '../../common/components/with_lazy_hook';
+
+export interface IntegrationsPageProps {
+  useAvailablePackages: AvailablePackagesHookType;
+}
+
+export const ConfigurationsIntegrationsHome = React.memo<IntegrationsPageProps>(
```

Comment:
Could double check with @bfishel about the view on a wide screen later (could do in a follow up PR):

Current layout flows to the left side:
<img width="2556" alt="Screenshot 2025-04-11 at 15 22 04" src="https://github.com/user-attachments/assets/1d252b3d-1a04-4598-99d4-061d11577958" />

Fleet page places it in the centre:
<img width="2549" alt="Screenshot 2025-04-11 at 15 22 12" src="https://github.com/user-attachments/assets/6a4b8ff5-2839-46ee-95b3-f2a3bb5ea526" />

Reply:
@bfishel double checked with her team and confirmed the left-align and max-width